store/src/java/com/zimbra/soap/ZimbraSoapContext.java

```diff
@@ -524,6 +526,11 @@ private void validateDelegatedAccess(Account targetAccount, DocumentHandler hand
             return;
         }
 
+        if ((handler != null) && handler instanceof DelegatableRequest
```

Comment:
is this required?

Reply:
Yes, this is required, so the proxyRequest was failing in this method on the receiver end. hence Stephan asked me to include this check so that request doesn't fail here. Post this fix proxyRequest was working fine